db/migrate/20230608075026_add_user_relations.rb

```diff
@@ -0,0 +1,20 @@
+class AddUserRelations < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :reviews, :user, foreign_key: true
+    add_reference :books, :user, foreign_key: true
+
+    default_user = User.create(email: 'default@mail.com', password: '123456')
```

Comment:
Since user's email cannot be duplicated
```suggestion
    default_user = User.find_or_create_by(email: 'default@mail.com') do |user|
      user.password = '123456'
    end
```